TESTS/mbed-crypto/sanity/main.cpp

```diff
@@ -51,7 +82,8 @@ utest::v1::status_t test_setup(const size_t number_of_cases)
 }
 
 Case cases[] = {
-    Case("mbed-crypto random", test_crypto_random, case_failure_handler)
+    Case("mbed-crypto random", test_crypto_random, case_failure_handler),
+    Case("mbed-crypto asymmetric encrypt/decrypt", test_crypto_asymmetric_encrypt_decrypt, case_failure_handler)
```

Comment:
Does it break anything to always include the comma at the end of the line here?

If we can always include commas at the end, the diffs will be more clean (won't look like we are adding new tests that were already added in a previous commit).

Reply:
I'm not sure I'll check if it breaks. I can also add a line break before the closing `}` of the cases array instead, then it will prevent the false appearance.